docs/sources/enterprise/team-sync.md

```diff
@@ -1,10 +1,9 @@
-+++
-title = "Team sync"
-description = "Grafana Team Sync"
-keywords = ["grafana", "auth", "documentation"]
-aliases = ["/docs/grafana/latest/auth/saml/"]
-weight = 1000
-+++
+---
+title: 'Team sync'
+description: 'Grafana Team Sync'
+keywords: ['grafana', 'auth', 'documentation']
```

Comment:
Unrelated to this PR, but does `auth` mean _authentication_, _authorization_, or both?

Reply:
Very good point. We have rules for authN and authZ in our wordlist but probably should add one for just auth.